chainer/distributions/independent.py

```diff
@@ -236,11 +236,11 @@ def _kl_independent_independent(dist1, dist2):
     # KL(p || q) and do a `reduce_sum` on the reinterpreted batch dimensions.
     if dist1.event_shape == dist2.event_shape:
         if p.event_shape == q.event_shape:
-            num_reduce_dims = len(dist1.event_shape) - len(p.event_shape)
-            reduce_dims = tuple([-i - 1 for i in range(0, num_reduce_dims)])
+            n_axes = len(dist1.event_shape) - len(p.event_shape)
+            n_reduce_dims = tuple([-i - 1 for i in range(0, n_axes)])
```

Comment:
Maybe I am mistaken but this is not a num but a tuple, right?

Reply:
Yes, but this PR is a backport of #8074, let us leave it as is here.

Reply:
Ok Lets do that and fix both in another PR :)